__tests__/app.test.js

```diff
@@ -127,50 +127,163 @@ describe("API's", () => {
+          .then(({ body }) => {
+            const { comments } = body;
+            expect(comments.length).toBe(11);
+            expect(comments).toBeSortedBy("created_at", { descending: true });
+          });
+      });
+      test("GET 404: Should return an error if the article_id is a valid datatype but does not exsist", () => {
+        return request(app)
+          .get("/api/articles/14/comments")
+          .expect(404)
+          .then(({ body }) => {
+            const { msg } = body;
+            expect(msg).toBe("Not Found");
+          });
+      });
+      test("GET 400: Should return an error if the article_id is no valid datatype or not included", () => {
+        return request(app)
+          .get("/api/articles/Invalid-Id/comments")
+          .expect(400)
+          .then(({ body }) => {
+            const { msg } = body;
+            expect(msg).toBe("Bad Request");
+          });
+      });
     });
-    test("GET 400: Should return an error if the article_id is no valid datatype", () => {
-      return request(app)
-        .get("/api/articles/Invalid-Id/comments")
-        .expect(400)
-        .then(({ body }) => {
-          const { msg } = body;
-          expect(msg).toBe("Bad Request");
-        });
+    describe("POST Requests", () => {
+      test("POST 201: Should add a comment for an article and returned the added comment", () => {
+        return request(app)
+          .post("/api/articles/1/comments")
+          .send({
+            username: "butter_bridge",
+            body: "Hello New Post",
+          })
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            comment.created_at = new Date(comment.created_at);
+            expect(comment).toMatchObject(
+              expect.objectContaining({
+                comment_id: 19,
+                body: "Hello New Post",
+                article_id: 1,
+                author: "butter_bridge",
+                votes: 0,
+              })
+            );
+            expect(comment).toMatchObject({ created_at: expect.any(Date) });
+          });
+      });
+      test("POST 201: Should add a comment for an article and return the added comment even if the body an empty string", () => {
+        return request(app)
+          .post("/api/articles/1/comments")
+          .send({
+            username: "butter_bridge",
+            body: "",
+          })
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            comment.created_at = new Date(comment.created_at);
+            expect(comment).toMatchObject(
+              expect.objectContaining({
+                comment_id: 19,
+                body: "",
+                article_id: 1,
+                author: "butter_bridge",
+                votes: 0,
+              })
+            );
+            expect(comment).toMatchObject({ created_at: expect.any(Date) });
+          });
+      });
```

Comment:
You could add in a test for a status 201: Ignores any unnecessary properties on the request body.